installation/phonesetup.md

```diff
@@ -0,0 +1,125 @@
+3. Reboot to TWRP to delete the Davlik and the normal Cache.
+
+## Applications - PoGo, RGC & PogoDroid
+
+### Pokémon Go
+
+Install the PoGo app from the Play Store and add PoGo to Magisk Hide via the magisk manager.
+
+>It's necessary to pass the Safetynet check to run the game on rooted phones. Check the Safetynet status in the MagiskManager App.
+
+### RGC
+
+RGC - Install [RGC (Remote GPS Controller)](https://github.com/Map-A-Droid/MAD/blob/master/APK/RemoteGpsController.apk)
+
+- RGC must be converted to a system app after it is installed.  There are two ways to do this, link2sd and Systemizer Magisk Module.  Start with Link2sd and then go to Systemizer if it doesnt work.
+
+- Install [Link2sd](https://play.google.com/store/apps/details?id=com.buak.Link2SD) to systemize RGC. Once installed go into Link2sd, you may be prompted to give it super user, if so do it.  Scroll down and find RGC.  Click on it, go to the menu and convert RGC to a system app.  It may ask you to reboot, if so do so.
+
+- If Link2sd didn't work try the Systemizer Magisk Module. You can find the download and instructions here [Systemizer Magisk Module](https://forum.xda-developers.com/apps/magisk/module-app-systemizer-t3477512)
+
+- In Android go into `Settings`, `Developer Options`.  Click on `Select mock location App`, select RGC.
+
+- Open RGC, grant it super user rights if asked.  Select allow for anything it asks access for.
+
+Configure RGC:
+
+#### Socket section
+
+- **Websocket URI** = `ws://ipofyourserver:8080` (Default port is 8080, you can change that in the config.ini)
+- **Websocket Origin** = pick a short name for your phone, this will need to match what you put in PogoDroid.  This Origin needs to be unique per running python instance.
+- **Auth** = optional, configure that via the mappings.
+
+#### Rooted devices section
+
+- **Reset GMS data** = Start with it Off and turn it to on if you need to troubleshoot Rubberbanding issues.
+- **Override OOM** = On. This will help to keep RGC running.
+
+#### Location Section
+
+- **Reset AGPS data continuously** = Off (turn this on when mock location is not used)
+- **Reset AGPS once** = Off (turn this on when mock location is not used)
+- **Use Android Mock Location** = On
+
+#### General Section
+
+- Start on Boot = On
+- Start RGC Delay = 30 (Play around with this setting. It's best practice to start RGC after PogoDroid to ensure that PogoDroid can inject properly.)
+- Start services on appstart = On
+
+### PogoDroid
+
+1. Install [PogoDroid](https://www.maddev.de/apk/PogoDroid.apk) (only necessary for MITM mode) on the phone.
+
+2. To login to PogoDroid you need to be a [Patreon supporter](https://www.patreon.com/user?u=14159560).  Become a supporter and link your account to Discord.  
+
+3. Click "Get Refresh Token", then when you get to the next page it will tell you to copy a password and API Token.  Copy both, put them in a notepad (if you dont have a notepad app, nows the time to get one).
```

Comment:
```suggestion
3. Click "Get Refresh Token", then when you get to the next page it will tell you to copy a password and API Token.  Copy both and put them in a notepad (if you don't have a notepad app, now is the time to get one).
```